### ISSUE TITLE
Wrong chapter shown as playing

### ISSUE THREAD

**Issue by mxsrm** [mattttvaughn/chronicle#12]
With one of the recent updates a bug was introduced, leading to the wrong chapter being shown as active in certain parts of Chronicle (not all).

Screenshots and video attached. Basically, the chapter shown in detail screen is always the one that was active before.

https://user-images.githubusercontent.com/7951225/139560564-028654a5-1d17-4a33-a131-47d29923236a.mp4
![Screenshot_20211030-234057](https://user-images.githubusercontent.com/7951225/139560578-0ccb556b-7ee2-4a28-8d0b-2618d9949ab0.png) ![Screenshot_20211030-234105](https://user-images.githubusercontent.com/7951225/139560579-f2b30d6a-0df8-4fb9-a4c3-4825fa9018f6.png)

**Comment by mattttvaughn** [comment]
Hi- thanks for writing up the bug, and for including the screenshots so I know what text is affected.

Is the problem here that the chapter never changes when the player changes chapters? (i.e. chapter on the player view will be set up correctly only when playback begins, but not if a new chapter begins)

If not, could you include steps to consistently reproduce this bug?

**Comment by mxsrm** [comment]
No, the display is always one chapter behind. It updates when the next one is starting, but to the one before. 

 Steps to reproduce:

Start any audio book
Wait for chapter change or manually change chapter by seeking forward or manually selecting a chapter

Note: My library/ books consists of audible audiobooks split into one pet m4a per chapter.

**Comment by mxsrm** [comment]
You can also see the bug really nice in the video from the opening post:

https://user-images.githubusercontent.com/7951225/139560564-028654a5-1d17-4a33-a131-47d29923236a.mp4

**Comment by mxsrm** [comment]
Any news on this? If you point me to the code sections that deal with this display I'm happy to look into it myself (java is a bit rusty, much more c++/rust experience in recent years).
